### PR TITLE
+mupdf

### PR DESCRIPTION
**Pull request by vcvvvc**
mupdf.com
https://github.com/ArtifexSoftware/mupdf
https://mupdf.readthedocs.io/en/latest/quick-start-guide.html

`pkg build` has an error, need help

### CODE REVIEW

File: projects/mupdf.com/package.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    x.org/xinerama: '*'
+    x.org/xrandr: '*'
+    mesa3d.org: '*'
+    gnome.org/glib: '*'
+    x.org/xtrans: '*'
+
+build:
+  dependencies:
+    git-scm.org: '*'
+    gnu.org/binutils: '*'
```
Comment (MarkusZoppelt) [inline projects/mupdf.com/package.yml]: You'll probably also need `gnu.org/gcc` for `g++`

```suggestion
    gnu.org/binutils: '*'
    gnu.org/gcc: '*'
```

**Comment by vcvvvc** [comment]
CXX build/release/source/fitz/tessocr.o

**Comment by vcvvvc** [comment]
muraster: error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory

**Comment by vcvvvc** [comment]
Hello, if this method fails the test, we can consider switching to this way.

I passed the local test, but I haven’t used pkgx to test yet.

https://mupdf.readthedocs.io/en/latest/using-mupdf.html 

```
gcc -I/usr/local/include -o example \
	test.c \
	/usr/local/lib/libmupdf.a \
	/usr/local/lib/libmupdf-third.a \
	-lm
```

**Comment by jhheider** [comment]
looks like it's good! thanks, @6923403 !

**Comment by vcvvvc** [comment]
> looks like it's good! thanks, @6923403 !

Cool, bro

**Pull request merged**